server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/DiversifiedSamplerIT.java

```diff
@@ -132,13 +132,14 @@ public void setupSuiteScopeCluster() throws Exception {
             client().prepareIndex("test")
                 .setId("" + i)
                 .setSource("author", parts[5], "name", parts[2], "genre", parts[8], "price", Float.parseFloat(parts[3]))
+                .setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE)
```

Comment:
This changes seem to be unnecessary, the `client().admin().indices().refresh(new RefreshRequest("test")).get();` would have done the same, no?

Reply:
The purpose of this is to refresh with each document and hopefully create enough segments to go down the concurrent search path (which from my local testing it did seem to do so). Currently it just refreshes once after indexing all the documents which I believe contributed to why this test was flaky instead of always failing.